src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -89,19 +148,21 @@ const RelocationBox = ({
 
   // A function that approves a relocation box submission and updates the backend state accordingly and then refetches all boxes (boxes can be approved from any tab)
   const approveRelocationBox = async () => {
+    const formData = getValues();
     await FYABackend.put('/boxHistory/update', {
       transactionID,
       boxID,
       status,
-      boxHolderName: boxHolderNameState,
-      boxHolderEmail: boxHolderEmailState,
-      zipCode: zipCodeState,
-      generalLocation: generalLocationState,
-      message: messageState,
-      launchedOrganically: launchedOrganicallyState,
+      boxHolderName: formData.name,
+      boxHolderEmail: formData.email,
+      zipCode: formData.zipcode,
+      country: formData.country.value,
+      generalLocation: formData.boxLocation,
+      message: formData.boxMessage,
+      launchedOrganically: formData.dropOffMethod === 'organic-launch',
     });
     // TODO: REPLACE US WITH COUNTRY INPUT
-    let coordinates = await getLatLong(zipCode, 'US');
+    let coordinates = await getLatLong(zipCode, formData.country.value);
```

Comment:
nit - Could we remove the TODO above this line?

Reply:
Ok, I modified it a little just in case the data is null in the country field, we can revert it once the database contains correct data since there's still a lot of boxes that have country as null and this might break the app.

Reply:
Okay, that looks good!